### ISSUE TITLE
Squaring does not remove extra nodes

### ISSUE THREAD

**Issue by cbeddow** [facebook/RapiD#815]
### Description

See this building: w1147172201

It has an unnecessary node that does not disappear, like it would when hitting the Q key to square a building in v1

### Version

2.0.0-beta.0

### What browser are you seeing the problem on? What version are you running?

Chrome v110.0

### The browser URL at the time you encountered the bug

https://mapwith.ai/rapid-v2-beta#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&id=w1147172201&map=19.53/25.40176/-101.01577

**Comment by Bonkles** [comment]
Chris- after you square the building, would you mind hitting 'refresh' on the browser and then when the 'keep changes' pop up appears, click yes to keep them. Do the extra unnecessary nodes disappear?

**Comment by cbeddow** [comment]
@Bonkles let me try with a new building. With this one, after I submitted the changeset, if i select and square it again, the node disappears, which is nice. 

Okay so--now I cannot reproduce. it may have had to do with lagging and I squared then moved away but the lag meant it didn't catch up to remove the node. Works when I got slowly now everywhere. Was thinking maybe when 2 nodes are very close together it happens, but no reproduce. So closing unless I encounter it again!
